### DISCUSSION TITLE
need help linkered proxy pod crashing

### DISCUSSION THREAD

**Discussion by kepark-dev** [linkerd/linkerd2#10264] [Help]
My linkered rpoxy pod is crashing. See information below:

Restart Count:  365
    Liveness:       http-get http://:9995/ping delay=10s timeout=1s period=10s #success=1 #failure=3
    Readiness:      http-get http://:9995/ready delay=0s timeout=1s period=10s #success=1 #failure=7
 
Events for that pod:
 
Events:
  Type     Reason     Age                     From     Message
  ----     ------     ----                    ----     -------
  Normal   Pulled     31m (x357 over 21h)     kubelet  Container image "cr.l5d.io/linkerd/controller:stable-2.11.5" already present on machine
  Warning  Unhealthy  15m (x3273 over 21h)    kubelet  Readiness probe failed: HTTP probe failed with statuscode: 502
  Warning  BackOff    5m56s (x4394 over 21h)  kubelet  Back-off restarting failed container
  Warning  Unhealthy  56s (x1098 over 21h)    kubelet  Liveness probe failed: HTTP probe failed with statuscode: 502
 
The container shows an Exit Code 137 that usually refers to a SYG_KILL: [Exit Codes in Containers & Kubernetes | Complete Guide | Komodor](https://urldefense.com/v3/__https:/komodor.com/learn/exit-codes-in-containers-and-kubernetes-the-complete-guide/__;!!BhdT!gdBDJ_6TXz8fL4RaVT6S8MvSkTb9A_e0mq0FiYun-nAa2YQ3OgmMm3-QOICbfI1HipziZ5P3p51Iu3An-uYTIex8oA0$)
 
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Error
      Exit Code:    137
 
Logs point to the same:
 
[ 77107.363401s]  INFO ThreadId(01) inbound:server{port=9995}:rescue{client.addr=10.3.8.63:52618}: linkerd_app_core::errors::respond: Request failed error=error trying to connect: Connection refused (os error 111)
[ 77117.363308s]  INFO ThreadId(01) inbound:server{port=9995}:rescue{client.addr=10.3.8.63:53982}: linkerd_app_core::errors::respond: Request failed error=error trying to connect: Connection refused (os error 111)

Can some one help on this issue.

**Comment by stevej** [comment]
Take a look at the logs for the `proxy-injector` for more context.
